railties/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+*   Remove default `X-Download-Options` header
 
+    This header is currently only used by Internet Explorer which
+    will be discontinued in 2022 and since Rails 7 does not fully
+    support Internet Explorer this header should not be a default one.
+
+    This change is a framework default from Rails 7.1.
```

Comment:
This line isn't needed, it's implicit from which part of the changelog it's in.

Reply:
Alright!